client/src/components/Header/Banner.jsx

```diff
@@ -1,6 +1,32 @@
+// import React from 'react'
```

Comment:
Everything looks  great   just  remove the commented code .

Reply:
@samuelkamotho92 done it. Check it now.